services/src/main/scala/cromwell/services/healthmonitor/impl/CommonMonitoredSubsystems.scala

```diff
@@ -0,0 +1,39 @@
+package cromwell.services.healthmonitor.impl
+
+import cromwell.core.docker.{DockerCliClient, DockerCliKey}
+import cromwell.services.EngineServicesStore
+import cromwell.services.healthmonitor.HealthMonitorServiceActor.{MonitoredSubsystem, OkStatus, SubsystemStatus}
+
+import scala.concurrent.{ExecutionContext, Future}
+
```

Comment:
Document this

Intention is to provide checks which are used by > 1 implementations